CMakeLists.txt

```diff
@@ -567,16 +576,17 @@ else() #!WIN32
         list(APPEND QUIC_COMMON_DEFINES DISABLE_POSIX_GSO)
     endif()
 
-    if (QUIC_ENABLE_SANITIZERS)
+    if (QUIC_ENABLE_SANITIZERS AND QUIC_ENABLE_LOGGING_STDOUT EQUAL 0)
         set(QUIC_ENABLE_LOGGING OFF)
         message(WARNING "LTTng logging is incompatible with sanitizers. Skipping logging")
     endif()
 
     if(QUIC_ENABLE_LOGGING)
-        if (APPLE)
-            set (QUIC_ENABLE_LOGGING OFF)
-            message(STATUS "Disabling all tracing on macos")
-            list(APPEND QUIC_COMMON_DEFINES QUIC_EVENTS_STUB QUIC_LOGS_STUB)
```

Comment:
Shouldn't we still be using stub if not stdout?

Reply:
The "Disabling all tracing on MacOS" stuff around line 288 sets `QUIC_ENABLE_LOGGING` to `OFF`, which means the stubs are being used.